### ISSUE TITLE
As a user, I should be able to navigate a book using the keyboard

### ISSUE THREAD

**Issue by respinos** [mlibrary/cozy-sun-bear#146]
And not by tabbing to controls.

- [ ] left/right arrow should de/advance through `flow`
- [ ] page up / page down should advanced through `scrolled-doc`

**Comment by respinos** [comment]
Addressed in `keyboard-and-preferences`
